config/style/aerialhybrid.json

```diff
@@ -0,0 +1,1315 @@
+{
+  "version": 8,
+  "name": "aerialhybrid",
+  "metadata": {
+    "maputnik:license": "https://github.com/maputnik/osm-liberty/blob/gh-pages/LICENSE.md",
+    "maputnik:renderer": "mbgljs",
+    "openmaptiles:version": "3.x"
+  },
+  "sources": {
+    "natural_earth_shaded_relief": {
+      "maxzoom": 2,
+      "tileSize": 256,
+      "tiles": ["https://klokantech.github.io/naturalearthtiles/tiles/natural_earth_2_shaded_relief.raster/{z}/{x}/{y}.png"],
+      "type": "raster"
+    },
+    "LINZ Basemaps": {
+      "type": "vector",
+      "url": ""
+    },
+    "LINZ-Imagery": {
+      "type": "raster",
+      "tileSize": 256,
+      "tiles": [],
+      "minzoom": 0,
+      "maxzoom": 28
+    },
+    "Nextzen-DEM": {
+      "type": "raster-dem",
+      "tiles": ["https://s3.amazonaws.com/elevation-tiles-prod/terrarium/{z}/{x}/{y}.png"],
+      "minzoom": 0,
+      "maxzoom": 14
+    }
+  },
+  "sprite": "https://maputnik.github.io/osm-liberty/sprites/osm-liberty",
```

Comment:
are we allowed to use these sprites/glyphs? do we use them elsewhere?

Reply:
I am not sure about these, why these are here.